src/ConfigurationInterfaces.ts

```diff
@@ -11,4 +11,5 @@ export interface PlatformClientOptions {
     environment?: Environment;
     region?: Region;
     responseHandlers?: ResponseHandler[];
+    fetchConfig?: RequestInit;
```

Comment:
I think this could use a clearer name. MDN describes the RequestInit parameter as
> An options object containing any custom settings that you want to apply to the request.

What do you think of `globalRequestSettings`? Since these settings will be applied to each request (hence `global`).

Also be sure to add some documentation in the README for this new option 😃

Reply:
Updating my PR to include this change